packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -343,6 +348,13 @@ export interface IGCRuntimeOptions {
      */
     runFullGC?: boolean;
 
+    /**
+     * Maximum session duration for a new container. If not present, a default value will be used.
```

Comment:
This is better than no comment, but it left me wondering if stating what the (current) default value is (30 days) would be worth it. I've had this "issue" with interfaces in TS instead of classes, where the default value for something is defined/applied in a different part of the code so it'd be easy for a comment here to go out of sync with the actual value... no airtight solution to that comes to mind. Just food for thought.

Reply:
Yeah, interesting point.  I think looking at both sides I would rather someone consult the code.  It's almost "wrong" to state on the interface that a default value would be used... but I think practically it's not risky of going out of date and may be helpful context.